plugins/rss/rss.php

```diff
@@ -318,20 +318,20 @@ public function __construct()
 		$this->version = 2;
 	}
 
-	public function add( $url, $hash, $timestamp )
+	public function add( $url, $hash, $timestamp, $guid = null, $title = null )
```

Comment:
Avoid default parameters. Make everything follow the new function prototype. This will address unintentional or undefined behavior.